applications/MPMApplication/mpm_application_variables.cpp

```diff
@@ -124,4 +124,13 @@ namespace Kratos
 
     // Stabilization variables
     KRATOS_CREATE_VARIABLE(int, STABILIZATION_TYPE)
+
+    // For friction
+    KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS( STICK_FORCE )
+    KRATOS_CREATE_VARIABLE(int, FRICTION_STATE)
```

Comment:
if there are only two states, then maybe you can reformulate the name and use a bool?

Reply:
I thought about it but ultimately decided on using an int instead of bool or enum, as it allows the FRICTION_STATE to be output and visualized in ParaView as categorical data. This is especially useful for debugging during development, but may also be beneficial for end users who want to confirm the friction state.